test/sequential/test-cluster-net-listen-ipv6only-none.js

```diff
@@ -16,9 +15,24 @@ const host = '::';
 const WORKER_ACCOUNT = 3;
 
 if (cluster.isMaster) {
-  const workers = new Map();
+  const workers = [];
 
-  const countdown = new Countdown(WORKER_ACCOUNT, () => {
+  for (let i = 0; i < WORKER_ACCOUNT; i += 1) {
+    const myWorker = new Promise((resolve) => {
+      const worker = cluster.fork().on('exit', common.mustCall((statusCode) => {
+        assert.strictEqual(statusCode, 0);
+      })).on('listening', common.mustCall((workerAddress) => {
+        assert.strictEqual(workerAddress.addressType, 6);
+        assert.strictEqual(workerAddress.address, host);
+        assert.strictEqual(workerAddress.port, common.PORT);
+        resolve(worker);
+      }));
+    });
+
+    workers.push(myWorker);
+  }
+
+  Promise.all(workers).then((resolvedWorkers) => {
```

Comment:
Likely a good idea to wrap the then handler in `common.mustCall()`

Reply:
Done!